### DISCUSSION TITLE
230514 - 앞으로 일주일간의 일정 공유

### DISCUSSION THREAD

**Discussion by jaynamm** [jaynamm/playground#21] [General]
앞으로 일주일 간의 일정에 대해서 정리해서 공유해보려고 합니다.
현재 일정은 이번 미니 프로젝트 기간에만 적용하려고 하고 그 이후에 대해서는 차후에 다시 회의를 진행하려고 합니다.
해당 일정에 대해서 수정해야할 부분이나 추가할 부분에 있어서는 댓글로 남겨주시면 감사합니다.

### 진행 사항
현재 프로젝트 진행 사항으로는 다음과 같습니다.
* 로그인 / 로그아웃 / 회원가입 기본 기능 구현 (스프링부트 / 리액트)
    * Main 화면에서 로그인 시 Home 화면으로 이동 후 피드가 보이도록 구현
    * 로그인 시 카카오 로그인 구현 진행중
* 피드 작성 / 피드 보기 (홈 화면) 기능 구현 (스프링부트 / 리액트)
    * 홈 화면에서 피드 작성 시 리다이렉션 후 화면에 보여주기
* 공지사항 작성 / 수정 / 삭제 기능 구현 (스프링부트)
    * 공지사항을 작성하는 화면은 현재 구현 중입니다. (리액트)
* QnA 게시판 기능 구현 진행중

### 해야할 일
앞으로 일정은 다음과 같이 생각하고 있습니다.
* 월요일 (5/15)
    * 용호님과 민우님의 휴가인 관계로 충분히 휴식을 취하고 수빈님이 합류하게 되면 현재 프로젝트 진행중인 깃 레포지토리를 공유할 예정입니다.
    * 다른 분들은 진행중인 개발 건에 대해서 진행해주시면 됩니다.
* 화요일 (5/16)
    * 전체적으로 모여서 회의를 진행하려고 합니다. 회의 내용으로는 다음과 같습니다.
        * 현재까지 개발한 기능 총 병합, 모든 기능들을 하나로 합치려고 합니다.
        * 앞으로의 개발 진행에 대한 토론
            기능 개발 건마다 PR 을 진행해서 바로바로 기능을 배포하려고 합니다.
        * 프로젝트 명세 정리
            각자 프로젝트를 진행하면서 공통적으로 정해야 하는 부분들에 대해서 정하고 
            각 기능에 대한 명세를 작성하고 각 기능 별로 담당을 다시 한번 확인하려고 합니다.
* 수요일 ~ 금요일 (5/17 ~ 5/19)
    * 수요일부터 각자 개발중인 기능에 추가로 개발하지 않고 지금까지 구현된 내용을 정리하는 시간이 필요할 것 같습니다.
    * 지금까지 개발한 기능들에 대해서 리펙토링을 진행하고 완성도를 조금이나마 높여가는 시간을 가지려고 합니다.
    * 발표일이 금요일에 정해진다면 현재까지 한 내용들에 대해서 정리하는 시간을 가지고 간단하게나마 PPT 자료를 정리하는 방향으로 잡아야할 것 같습니다.
    * 생각보다 시간이 많지 않아 현재까지 한 내용들을 정리하는 방향으로 잡아도 좋을 것 같다는 생각이 듭니다.
    * 발표에 대해서도 시간을 내어 회의를 진행해야 합니다.

<br>

현재 해야할 일이 굉장히 많아보이기 때문에 욕심내지 않으려고 합니다.
현재 미니 프로젝트 기간이기 때문에 기간 내에 할 수 있는 만큼만 진행해서 마무리하려고 합니다.  
미니 프로젝트 이후 계속해서 개발을 진행할 예정이고 최종 프로젝트까지 갈 예정이기 때문에 길게 보고  
각자 원하는 결과를 얻어갈 수 있었으면 합니다.

<br>

위의 일정에 대한 의견을 남겨주시면 감사합니다!!!

**Comment by KKYHH** [comment]
# 넘모 좋습니다요

**Reply by KKYHH** [reply]
그런데 15일 휴가 용호 민우 아닌가용? 광일님 월또휴?!

**Reply by jaynamm** [reply]
앗 수정수정~~ ㅋㅋㅋ

**Comment by moneyless1989** [comment]
>월요일 (5/15)
용호님과 광일님의 휴가인 관계로 충분히 휴식을 취하고 수빈님이 합류하게 되면 현재 프로젝트 진행중인 깃 레포지토리를 공유할 예정입니다.

저 월요일에 나옵니다! 다른 분이랑 착각하신 것 같아요.

> 피드 작성 / 피드 보기 (홈 화면) 기능 구현 (스프링부트 / 리액트)
홈 화면에서 피드 작성 시 리다이렉션 후 화면에 보여주기

일단 금요일에 댓글 API 기능 확인은 해놨는데... 화요일에 같이 보고 나서 통합을 검토해야 할 것 같네요.

**Reply by jaynamm** [reply]
민우로 수정했음 ㅋㅋㅋ
오키 그럼 화요일에 다같이 확인하고 PR 진행하도록 합시다!

**Reply by KKYHH** [reply]
# 규웃

**Comment by jaynamm** [comment]
미니 프로젝트에 굳이 발표자료를 만들지 말고 README 파일을 잘 정리해서 보여주면 되지 않을까 생각도 듭니다.
그리고 직접 화면을 띄워 보여주면 되지 않을까 합니다.. 흠흠..

**Reply by KKYHH** [reply]
# 아주 딱
